### DISCUSSION TITLE
Namespace management using CRDs.

### DISCUSSION THREAD

**Discussion by sriabhi02nov** [alexandrevilain/temporal-operator#155] [Q&A] [answered]
Hi Alexandre,

Do you have any reference how i can create namespace in  temporal via CRD resource and from where i will get more info about this CRD resource.

Thanks in advance.

**Comment by alexandrevilain** [comment, marked as answer]
Hi! 

Here is the api documentation: https://github.com/alexandrevilain/temporal-operator/blob/main/docs/api/v1beta1.md#temporal.io/v1beta1.TemporalNamespace

And you can find an example here: https://github.com/alexandrevilain/temporal-operator/blob/main/examples/cluster-postgres/03-temporal-namespace.yaml 

Hope it answers your question 🙂

**Reply by sriabhi02nov** [reply]
Yes, thanks . I was able to get it from examples you mentioned in postgres. 
I was working with cassandra so was looking into the same.

Thanks for inputs.
